docs/source/releasenotes/0.13.1.rst

```diff
@@ -0,0 +1,181 @@
+  instead in order to more easily handle the common case of specifying a uniform
+  spatial distribution over a sphere (also see the new
+  :func:`openmc.stats.spherical_uniform` function).
+
+- If you are building OpenMC from source, note that several of our CMake options
+  have been changed:
+
+  ==========  ======================
+  Old option  New option
+  ==========  ======================
+  debug       ---
+  optimize    ---
+  profile     OPENMC_ENABLE_PROFILE
+  coverage    OPENMC_ENABLE_COVERAGE
+  openmp      OPENMC_USE_OPENMP
+  ---         OPENMC_USE_MPI
+  dagmc       OPENMC_USE_DAGMC
+  libmesh     OPENMC_USE_LIBMESH
+  ==========  ======================
+
+  The ``debug`` and ``optimize`` options have been removed; instead, use the
+  standard `CMAKE_BUILD_TYPE
+  <https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html>`_
+  variable.
+
+------------
+New Features
+------------
+
+- Two new composite surfaces: :class:`openmc.model.IsogonalOctagon` and
+  :class:`openmc.model.CylinderSector`.
+- The :class:`~openmc.DAGMCUniverse` class now has a
+  :attr:`~openmc.DAGMCUniverse.bounding_box` attribute and a
+  :meth:`~openmc.DAGMCUniverse.bounding_region` method.
+- When translating a :class:`~openmc.Region` using the
+  :meth:`~openmc.Region.translate` method, there is now an ``inplace`` argument.
+- The :class:`~openmc.Material` class has several new methods and attributes:
+
+  - The :meth:`~openmc.Material.add_components` methods allows you to add
+    multiple nuclides/elements to a material with a single call by passing a
+    dictionary.
+  - The :meth:`~openmc.Material.get_activity` method returns the activity of a
+    material in Bq, Bq/g, or Bq/cm³.
+  - The :meth:`~openmc.Material.remove_element` method removes an element from a
+    material
+  - The :meth:`~openmc.Material.get_nuclide_atoms` method gives the number of
+    atoms of each nuclide in a material
+
+- All mesh classes now have a ``volumes`` property that provides the volume of
+  each mesh element as well as ``write_data_to_vtk`` methods.
+- Support for externally managed MOAB meshes or libMesh meshes.
+- Multiple discrete distributions can be merged with the new
+  :meth:`~openmc.stats.Discrete.merge` method.
+- The :func:`openmc.stats.spherical_uniform` function creates a uniform
+  distribution over a sphere using the
+  :class:`~openmc.stats.SphericalIndependent` class.
+- Univariate distributions in the :mod:`openmc.stats` module now have
+  ``sample()`` methods.
+- An ``openmc_sample_external_source`` function has been added to the C API with
+  a corresponding Python binding :func:`openmc.lib.sample_external_source`.
+- The track file generation capability has been completely overhauled. Track
+  files now include much more information, and a new :class:`~openmc.Tracks`
+  class allows access to track file information from the Python API. Multiple
+  tracks are now written to a single file (one per MPI rank).
```

Comment:
```suggestion
  files now include much more information, and a new :class:`~openmc.Tracks`
  class allows access to track file information and a ``write_tracks_to_vtk``
  method from the Python API. Multiple tracks are now written to a single file
  (one per MPI rank).
```

Reply:
Thanks for the suggestion -- I added something very close to this. Also, it occurred to me that "tracks" in the method `Tracks.write_tracks_to_vtk` is a little superfluous, so I renamed the method `write_to_vtk`. Hope you're ok with that!

Reply:
Looks super, thanks for getting that in